### ISSUE TITLE
feature request: support access to special folder "camera roll"

### ISSUE THREAD

**Issue by tipa** [OneDrive/onedrive-sdk-csharp#144]
Currently it seems only the "AppRoot" special folder is supported.
Something like this would be great: OneDriveClient.Drive.Special.CameraRoll.(...)

**Comment by cdmayer** [comment]
Fixed in #146. You may need to download the new code from Github as it won't be on Nuget right away.

**Comment by tipa** [comment]
Is there an ETA of a new NuGet package?
I would wait if it was some more days but I would get the GitHub code if there is no date in sight...

**Comment by cdmayer** [comment]
Not currently. Could be a couple weeks at this point.

**Comment by tipa** [comment]
I saw the new NuGet package v1.2.0 today but I am surprised that this small fix isnt included. Or did I miss something?

**Comment by cdmayer** [comment]
Sorry for the delay. This has been fixed in V2.0 for a while now. Just do:

```csharp
var cameraRoll =  await client.Drive.Special.CameraRoll.Request().GetAsync();
```

**Comment by tipa** [comment]
I am using version 2.0.2 for UWP and CameraRoll or other special folders **seem not to be available**.
![unbenannt](https://cloud.githubusercontent.com/assets/2863198/19386913/6553722a-9217-11e6-828e-14a2c5604806.png).

I currently have to use the following workaround:
`var request = OneDriveClient.Drive.Special.CameraRoll.Request().Select("name,parentReference");
 request = new ItemRequest(request.RequestUrl.Replace("root", "special/cameraroll"), request.Client, request.QueryOptions);
`

**Comment by cdmayer** [comment]
Apologies, you are correct. The code is present, but was not added to the solution as a file to be compiled, so the methods are not available (as you pointed out). I will get a PR up today to fix this. Sorry about that!

**Comment by tipa** [comment]
great to hear! when will it be on nuget?

**Comment by cdmayer** [comment]
Fixed in #182, nuget here: https://www.nuget.org/packages/Microsoft.OneDriveSDK/2.0.3
